src/st/no0/e_magically_sealed_door.c

```diff
@@ -0,0 +1,6 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+#include "no0.h"
+
+#define MAGICALLY_SEALED_DOOR_PALETTE 0x245
```

Comment:
NZ0 and NO0 use different palette values here (and by extension clut) so I've added this so the stage can override the palette if required. 

Can update this to be inline #ifdef conditionals in the `.h` based on the stage if that is preferable